app/controllers/idv/verify_info_controller.rb

```diff
@@ -392,11 +392,23 @@ def add_proofing_costs(results)
         elsif stage == :threatmetrix
           # transaction_id comes from request_id
           tmx_id = hash[:transaction_id]
+          log_irs_tmx_fraud_check_event(hash) if tmx_id
```

Comment:
Makes sense to me. Cc @soniaconnolly is there a VerifyStep left, or equivalent we would also want to add this instrumentation in? Or has that already been removed?

Reply:
The unsupervised VerifyStep is deleted, but there is still an [in-person VerifyStep](https://github.com/18F/identity-idp/blob/main/app/services/idv/steps/in_person/verify_step.rb).

Reply:
IPP is out of scope for Attempts API for now, so there's just that flow and this one, right? We might want to make a comment or note in that class as a TODO/FIXME equivalent, but maybe not add the logic changes yet

Reply:
IPP and unsupervised are the only Verify steps I know about. I'm in process of replacing the in-person FSM VerifyStep with a separate controller.